lib/src/geo/crs.dart

```diff
@@ -29,10 +29,15 @@ abstract class Crs {
     this.wrapLat,
   });
 
+  // Project a spherical LatLng coordinate into planar space (unscaled).
   Projection get projection;
 
-  /// Converts a point on the sphere surface (with a certain zoom) in a
-  /// map point.
+  // Scale planar coordinate to scaled map point.
+  (double, double) transform(double x, double y, double scale);
```

Comment:
Perhaps this could take and return a `DoublePoint(x, y)`? It would make it more convenient to use, although slightly less versatile.

Reply:
I will close this simply because you opened another discussion on PointDouble and I'd like to consolidate.